static/js/components/dashboard/CourseListCard.js

```diff
@@ -39,22 +42,30 @@ export default class CourseListCard extends React.Component {
     showStaffView: boolean
   }
 
-  handleCalculatePriceClick = (e: Event) => {
-    const { openFinancialAidCalculator } = this.props
-    if (openFinancialAidCalculator) openFinancialAidCalculator()
-    e.preventDefault()
-  }
-
-  renderPriceMessage(): ?React$Element<*> {
-    const { program } = this.props
-
+  renderGradesOutOfDateMessage(): ?React$Element<*> {
+    const { program, email } = this.props
+    const hasSocialAuthMessage = program.has_socialauth_for_backend
+      ? "Please visit"
+      : `Please use your current email ${email ? email : ""} to login to`
     return (
-      <p className={priceMessageClassName}>
-        To get credit for the courses in this program, you must pay for a
-        verified certificate from {programBackendName(program)}. If you want to
-        audit courses for FREE and upgrade later, click Enroll then choose the
-        audit option.
-      </p>
+      <div className="callout callout-warning">
+        <img src="/static/images/c-warning-1.svg" alt="Warning" />
+        <div>
+          The following course, enrollment, and grade information is outdated.
+          <br />
```

Comment:
Is this line break necessary here? I think the message would flow better in one single paragraph.

Reply:
The break was in the design, but I can remove it.